### PR TITLE
Add 1.5 vs 1.6 differences in hosted kubeadm

### LINKED ISSUE #677: Fix kubeadm install requirements on master label
The docs at `getting-started/kubernetes/installation/hosted/kubeadm/` have requirements and a verification that only works for k8s 1.5.  It needs to be updated to include the proper label for 1.6.

### PR DESCRIPTION
**Pull request by tmjd**
Fixes #677

### CODE REVIEW

**Comment by djosborne** [comment]
Kubeadm completely rewrote their versioning scheme. See https://github.com/projectcalico/calico/issues/648#issuecomment-291266588

Should just read "kubeadm 1.6" and "kubeadm 1.5"

File: master/getting-started/kubernetes/installation/hosted/kubeadm/index.md [hunk 1]
```diff
@@ -44,13 +44,25 @@ automatically on the master when using kubeadm.
 
 To check if the required label is applied, run the following command:
 
+For Kubeadm <=1.6.0-alpha with Kubernetes 1.5.x:
+
 ```shell
 $ kubectl get node <master_name> -o yaml | grep kubeadm
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: Should we really be showing this at all? Feels a little out of scope to me....

**Comment by tmjd** [comment]
@djosborne PTAL
I reduced the label check down to just providing info on retrieving the master info.  If you think we should just rid of it all together I'm fine with that too.

**Comment by djosborne** [comment]
Yeah I pretty strongly feel that it's unnecessary for us to be disclosing this in our docs. 
1. Users should know how to check it themselves
2. Users shouldn't need to check it for Calico installation

**Comment by tmjd** [comment]
@djosborne After our chat, is there anything else to change here since you are good with providing the means to check the label?

**Comment by djosborne** [comment]
Can you back apply just to v2.1?

Then merge

**Pull request merged**